src/logger/logger.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+  use(req: Request, res: Response, next: NextFunction) {
+    const ip = req.ip || req.connection.remoteAddress;
+    console.log(`IP Address: ${ip}`);
+    next();
+  }
+}
```

Comment:
Good job!
Better to name it as "LogIpMiddleware" and store it in src/middleware/log-ip.middleware.ts